src/server/utils/executor/executor.ts

```diff
@@ -109,26 +114,26 @@ export function externalFactory(dataSource: DataSource, druidRequester: Requeste
   };
 
   if (dataSource.introspection === 'none') {
-    return Q(External.fromJS({
-      engine: 'druid',
+    return Q(new DruidExternal({
+      suppress: true,
       dataSource: dataSource.source,
       timeAttribute: dataSource.timeAttribute.name,
       customAggregations: dataSource.options['customAggregations'],
-      attributes: deduceAttributes(dataSource),
+      attributes: AttributeInfo.applyOverrides(deduceAttributes(dataSource), dataSource.attributeOverrides),
```

Comment:
I have a habit of doing that whenever the thing I am accessing `dataSource.options` does not have a strict type. I think this used to be a TypeScript limitation way back in the day... probably does not make sense any more.